atomic_reactor/util.py

```diff
@@ -5,7 +5,7 @@
 This software may be modified and distributed under the terms
 of the BSD license. See the LICENSE file for details.
 """
-
```

Comment:
Newline should stay, but I need little bit of explanation why `_hashlib` ?

Reply:
I wanted to describe the type of return value from hashlib methods like `hashlib.md5()` or `hashlib.sha256()`. `type(returned object)` returns `_hashlib.HASH`.

Reply:
The newline is back.